test/mocha.opts

```diff
@@ -1,3 +1,4 @@
---timeout 10000
 --require ./test/setup.js
---compilers js:babel-register
```

Comment:
We might still need this in case we want to run mocha on the source code without nyc.